util/detectOS.ts

```diff
@@ -1,7 +1,7 @@
 import type { UserOS } from '@/types/userOS';
 
 export const detectOsInUserAgent = (userAgent: string | undefined): UserOS => {
-  const osMatch = userAgent?.match(/(Win|Mac|Linux)/);
+  const osMatch = userAgent?.match(/(Win|Mac|Linux|AIX)/);
```

Comment:
I think as mentioned before, instead of adding `AIX` to this match, we should remove the "case AIX"